src/Service.Tests/CosmosTests/SamplerTests.cs

```diff
@@ -206,8 +219,7 @@ Mock<TimeBasedSampler> timeBasedSampler
         [TestCleanup]
         public async Task CleanUp()
         {
-            await _containerWithIdPk.DeleteContainerAsync();
-            await _containerWithNamePk.DeleteContainerAsync();
+            await _database.DeleteAsync();
```

Comment:
just to confirm, this doesn't run after each datarow, right?

Reply:
no..it run only once.